examples/hello-world.erln.md

```diff
@@ -0,0 +1,30 @@
+# A trivial experiment
+```{dot !}
+input [data="Hello, World!"];
+input -> log;
+```
+
+# Specifying options
+```{js !!}
+exp.options.input = {data: 'Hello, World!'};
```

Comment:
Instead of `exp` we could put `options`, `aliases`, `imports` in the scope directly.